### PR TITLE
Add list of integrations to CSE Automation Service Beta doc

### PR DESCRIPTION
**Pull request by pkazmir-sumo**
Added list of integrations to Beta doc

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/integrations/automation-service.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Some notes about Playbooks, Integrations and Actions in the Beta that may differ
 * Playbooks type must be **CSE**.
 * The CSE Automation Service only supports automated Enrichment, Notification, and Custom Action types at this time. 
 * Actions can run "on premise" via a Bridge or can run directly through the Cloud. For security and performance reasons, only Certified Integrations/Actions can run directly through the Cloud; custom Actions must run "on premise".
-* Cloud SOAR App Central, where you can browse the full Integration and Playbook catalog, is not yet connected to the CSE Automation Service. Contact Sumo Logic if you are interested in a specific Integration that is not yet included with the CSE Automation Service.
+* Cloud SOAR App Central, where you can browse the full Integration and Playbook catalog, is not yet connected to the CSE Automation Service. A selection of popular Integrations have been added to your environment automatically, but the full list of available Integrations is included at bottom of this document. Contact your Sumo Logic account representative if you would like to have one of these Integrations added to your environment, or if you're interested in an Integration that's not listed.
```
Comment (jpipkin1) [inline docs/cse/integrations/automation-service.md:81]:
```suggestion
* Cloud SOAR App Central, where you can browse the full Integration and Playbook catalog, is not yet connected to the CSE Automation Service. A selection of popular Integrations have been added to your environment automatically, but the full list of available Integrations is included in [Available Integrations](#available-integrations) below. Contact your Sumo Logic account representative if you would like to have one of these Integrations added to your environment, or if you're interested in an Integration that's not listed.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Looks good, @pkazmir-sumo. I made one suggestion to add a link that points to the new section. 

Although the table looks good, I'm concerned about being able to add new integrations to it later. Wouldn't it better to simply have a long list?

**Review by pkazmir-sumo — COMMENTED** [review]
Accepted the first change, revised the doc to turn the table into (literally) a list.

**Review by jpipkin1 — APPROVED** [review]
It looks great, @pkazmir-sumo. I approve the changes.

Later on I can see the "Available Integrations" could be its own article. But for the purposes of beta, this works well. Thanks for adding the list.

**Pull request merged**